_includes/markdown/Tools.md

```diff
@@ -20,7 +20,9 @@ At 10up, we use [Vagrant](https://www.vagrantup.com/) and/or [Docker](https://ww
 
 [Bower](https://bower.io/) - A good tool to manage front end packages. Usually everything we need is bundled with WordPress. Sometimes we need something like "Chosen.js" that isn’t included. Bower is a good way to manage external libraries like that but is not necessary on most projects.
 
-[Composer](https://getcomposer.org) - We use Composer for managing PHP dependencies. Usually everything we need is bundled with WordPress. Sometimes we need external libraries like "Patchwork". Composer is a great way to manage those external libraries but is not necessary on most projects.
+[Composer](https://getcomposer.org) - We use Composer for managing PHP dependencies. Usually everything we need is bundled with WordPress, but sometimes we need external PHP libraries like "Patchwork". Composer is a great way to manage those external libraries.
+
+WordPress plugins in a project should be managed by Composer. Plugin versions can be easily managed in a single composer.json, helping ensure plugin versions between environments are in sync. [WordPress Packagist](https://wpackagist.org/) provides a Composer repository that mirrors all public WordPress plugins and themes.
```

Comment:
I think we should be a _little_ careful about the language we use here. Don't get me wrong, I am all about managing plugins and even WordPress itself via Composer, but it's not the right call for every project. Having the blanket statement of 'WordPress plugins in a project should be managed by Composer' is pretty heavy handed. I would say something more along the lines of 'When a WordPress install is managed and maintained by an engineering team, and when the infrastructure supports it, plugins in a WordPress project should be managed by Composer'.